lib/ansible/utils/module_docs_fragments/kubernetes.py

```diff
@@ -0,0 +1,116 @@
+    - present
+    - absent
+  force:
+    description:
+    - If set to C(True), and I(state) is C(present), an existing object will be updated,
+      and lists will be replaced, rather than merged.
+    default: false
+    type: bool
+  resource_definition:
+    description:
+    - "Provide a valid YAML definition for an object when creating or updating. NOTE: I(kind), I(api_version), I(name),
+      and I(namespace) will be overwritten by corresponding values found in the provided I(resource_definition)."
+    aliases:
+    - definition
+    - inline
+  src:
+    description:
+    - "Provide a path to a file containing a valid YAML definition of an object to be created or updated. Mutually
+      exclusive with I(resource_definition). NOTE: I(kind), I(api_version), I(name), and I(namespace) will be
+      overwritten by corresponding values found in the configuration read in from the I(src) file."
+  api_version:
+    description:
+    - Use to specify the API version when deleting an object, or when creating an object, such as namespace or
+      project, that only requires a I(name) attribute.
+    aliases:
+    - api
+    - version
+  kind:
+    description:
+    - Use to specify the object kind when deleting an object, or when creating an object, such as namespace or
+      project, that only requires a I(name) attribute.
+  name:
+    description:
+    - Use to specify an object name when deleting an object, or when creating an object, such as namespace or
+      project, that only requires a name attribute.
+  namespace:
+    description:
+    - Use to specify the namespace or project when deleting an object, or when creating an object, such as namespace
+      or project, that only requires a I(name) attribute.
+  host:
+    description:
+    - Provide a URL for accessing the API. Can also be specified via K8S_AUTH_HOST environment variable.
+  api_key:
+    description:
+    - Token used to authenticate with the API. Can also be specified via K8S_AUTH_API_KEY environment variable.
+  kubeconfig:
+    description:
+    - Path to an existing Kubernetes config file. If not provided, and no other connection
+      options are provided, the openshift client will attempt to load the default
+      configuration file from I(~/.kube/config.json). Can also be specified via K8S_AUTH_KUBECONFIG environment
+      variable.
+  context:
+    description:
+    - The name of a context found in the config file. Can also be specified via K8S_AUTH_CONTEXT environment variable.
+  username:
+    description:
+    - Provide a username for authenticating with the API. Can also be specified via K8S_AUTH_USERNAME environment
+      variable.
+  password:
+    description:
+    - Provide a password for authenticating with the API. Can also be specified via K8S_AUTH_PASSWORD environment
+      variable.
+  cert_file:
+    description:
+    - Path to a certificate used to authenticate with the API. Can also be specified via K8S_AUTH_CERT_FILE environment
```

Comment:
These are specific env vars created for ansible?  I assume there's a spec that covers this - we have explicitly not added them to Kube for reasons, but they seem fine to use from ansible.

Reply:
@smarterclayton 

The environment vars are introduced by the [Openshift client](https://github.com/openshift/openshift-restclient-python). We're making them available here for convenience.